packages/core/src/routes/interaction/verifications/profile-verification.ts

```diff
@@ -177,7 +177,7 @@ const profileExistValidation = async (
 export default async function profileVerification(
   ctx: InteractionContext,
   identifiers: Identifier[]
-) {
+): Promise<Profile | undefined> {
```

Comment:
Wondering why we want to return the `profile`? Since we did not modify the `profile` in this method,

Reply:
keep the same function behavior with other verification methods. Should use the returned value instead of directly accessing the ctx.  Will refactor it in the next PR for better understanding.